cleanlab/object_detection/summary.py

```diff
@@ -18,9 +18,14 @@
 from typing import Optional, Any, Dict, Tuple, Union, TYPE_CHECKING, TypeVar
```

Comment:
Could you call this file dataset.py, consistent with all the other modules?
for example: 

https://github.com/cleanlab/cleanlab/tree/master/cleanlab/multilabel_classification

Reply:
We *cannot* just move existing code like this, it breaks the code for users of version 2.5.0. 

@Steven-Yiran Sorry about the confusion, please move all the methods back into summary.py

Reply:
sounds good